.travis.yml

```diff
@@ -1,23 +1,21 @@
 language: python
 cache: pip
 python:
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-  - "nightly" # currently points to 3.6-dev
+- '2.7'
+- '3.3'
+- '3.4'
+- '3.5'
```

Comment:
3.7 - 3.8 ?

Reply:
Good point this year

Reply:
sorry, I noticed your PR only today.

Reply:
i#ll take a look when i recovered a bit more from he cold, right now im mush